MeowWidget/MeowWidget.swift

```diff
@@ -25,6 +25,7 @@ struct MeowWidgetEntry: TimelineEntry {
 }
 
 struct Provider: TimelineProvider {
+    @AppStorage("widgetRefreshInterval") var refreshInterval: Int = 10
```

Comment:
应对 UserDefaults 键名使用大写开头。

```suggestion
    @AppStorage("WidgetRefreshInterval") var refreshInterval: Int = 10
```